spec/forms/idv/phone_form_spec.rb

```diff
@@ -142,7 +142,7 @@
       context 'including country' do
         let(:previous_params) { { phone: '2255555000', international_code: 'IE' } }
         it 'uses the previously submitted phone + country as the initial values' do
-          expect(subject.phone).to eq('+3532255555000')
+          expect(subject.phone).to eq('+353 225 555 5000')
```

Comment:
looks like we get new pretty formatting for Irish numbers now, so I figured we'd accept the change, since this is what happens to US numbers